src/Progress/Progress.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {useEffect, useRef, useState, forwardRef, memo} from 'react'
+import {View, StyleSheet, Animated, Dimensions} from 'react-native'
+
+interface IProgressProps {
+  /**
+   * Current percent of the Progress
+   * default 0, max 100
+   */
+  value: number
```

Comment:
the value could be null or undefined?

Reply:
Already added default value = 0 in props